README.md

```diff
@@ -108,8 +108,7 @@ The current test suite is limited. Tests can be run with `make test`.
 ## Acknowledgements
 
 This project was originally forked from [Tom Hawkin's Verilog parser]. While the
-front end has changed substantially in adding SystemVerilog support, his project
-was as a great starting point.
+front end has changed substantially to parse a different language, his project was a great starting point.
```

Comment:
Can you restore the 80 character wrapping here? I believe "was" should start on the next line.